prow/config/config.yaml

```diff
@@ -259,6 +259,20 @@ presubmits:
           value: "TRUE"
         image: registry.hub.docker.com/library/golang:1.13.7
         imagePullPolicy: Always
+  - name: gomod
```

Comment:
It looks like this would only apply to the BMO, right? Could you update the commit message to indicate that?

Reply:
Done